audits/manifest/background-color.js

```diff
@@ -26,13 +26,16 @@ class ManifestBackgroundColor {
     return 'Contains background_color';
   }
 
-  static audit(inputs) {
-    let hasBackgroundColor = false;
-    const manifest = inputs.manifest;
+  static hasBackgroundColorValue(manifest) {
+    /* eslint-disable camelcase */
+    return Boolean(manifest !== undefined &&
```

Comment:
I can drop the boolean cast and go back to the !! cast we had before, nbd I find Boolean to be a touch more readable